integrations/huggingface-transformers/README.md

```diff
@@ -62,129 +96,207 @@ output:
+        - classifier
+
+distillation_modifiers:
+  - !DistillationModifier
+     hardness: eval(distill_hardness)
+     temperature: eval(distill_temperature)
+     distill_output_keys: [logits]
+
+constant_modifiers:
+  - !ConstantPruningModifier
+      start_epoch: 0.0
+      params: __ALL_PRUNABLE__
+
+regularization_modifiers:
+  - !SetWeightDecayModifier
+      start_epoch: 0.0
+      weight_decay: eval(weight_decay)
+```
 
-The following command will use the 80% sparse-quantized BERT model from the SparseZoo and fine-tune it on the SQuAD dataset, resulting in a model that achieves an F1 of 88.5% on the validation set. Keep in mind that the `--distill_teacher` argument is set to pull a dense SQuAD model from the SparseZoo to enable it to run independent of the dense teacher step. If you trained a dense teacher, change this out for the path to your model folder:
 
+</details>
+
+Run the following to sparse transfer learn the 90% pruned BERT model on the SST2 dataset:
 ```bash
-sparseml.transformers.question_answering \
-    --output_dir models/sparse_quantized \
-    --model_name_or_path zoo:nlp/masked_language_modeling/bert-base/pytorch/huggingface/wikipedia_bookcorpus/12layer_pruned80_quant-none-vnni \
-    --recipe zoo:nlp/masked_language_modeling/bert-base/pytorch/huggingface/wikipedia_bookcorpus/12layer_pruned80_quant-none-vnni?recipe_type=transfer-question_answering \
-    --distill_teacher zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/base-none \
-    --dataset_name squad \
-    --per_device_train_batch_size 12 \
-    --per_device_eval_batch_size 24 \
-    --preprocessing_num_workers 6 \
-    --do_train \
-    --do_eval \
-    --evaluation_strategy epoch \
-    --fp16 \
-    --seed 21636  \
-    --per_device_train_batch_size 16 \
-    --per_device_eval_batch_size 24 \
-    --preprocessing_num_workers 6 \
-    --save_strategy epoch \
-    --save_total_limit 1
+sparseml.transformers.text_classification \
+  --output_dir pruned_quantized_obert-text_classification_sst2 \
+  --model_name_or_path zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/pruned90-none \
+  --recipe zoo:nlp/sentiment_analysis/obert-base/pytorch/huggingface/sst2/pruned90_quant-none \
+  --distill_teacher zoo:nlp/sentiment_analysis/obert-base/pytorch/huggingface/sst2/base-none \
+  --task_name sst2 --max_seq_length 128 --per_device_train_batch_size 32 --per_device_eval_batch_size 32 --preprocessing_num_workers 6 \
+  --do_train --do_eval --evaluation_strategy epoch --fp16  \
+  --save_strategy epoch --save_total_limit 1
 ```
 
-### Exporting to ONNX
+The script uses the SparseZoo stubs to identify and download the starting checkpoint and YAML-based recipe file from the SparseZoo. SparseML parses the transfer learning recipe and adjusts the trainign process to maintain sparsity during the fine-tuning process.
+
+The resulting model is 90% pruned and quantized, and achieves 92% validation accuracy on SST2!
```

Comment:
The resulting model is 90% pruned and quantized, achieving 92% validation accuracy on SST2!